docker/minio.env

```diff
@@ -2,9 +2,8 @@ ACCESS_KEY=
 SECRET_ACCESS_KEY=
 STORAGE_PROVIDER=
 BUCKET_NAME=
-FOLDER_NAME=
```

Comment:
is this a breaking change?

Reply:
shouldn't be, they can still set folder name (although you prefer they will set workspace instead)